Makefile

```diff
@@ -316,7 +325,7 @@ CI_JS_SUITES := doctool inspector known_issues message parallel pseudo-tty seque
 
 # Build and test addons without building anything else
 test-ci-native: LOGLEVEL := info
-test-ci-native: | test/addons/.buildstamp
+test-ci-native: | test/addons/.buildstamp test/addons-napi/.buildstamp
```

Comment:
This may actually be the only relevant change, but I updated the target as well to reflect the corresponding `test/addons/.buildstamp` target.